src/parsers/parser.ts

```diff
@@ -15,6 +15,16 @@ export type EncodeArgs<Value> = {
    * The value to encode.
    */
   value: Value;
+
+  /**
+   * Whether to use the non-standard packed mode.
+   */
+  packed: boolean;
+
+  /**
+   * Whether to use tight packing mode. Only applicable when `packed` is true.
+   */
+  tight: boolean;
```

Comment:
We pass this around, but it is not used? 🤔

Reply:
It's used in the `array` parser.